jp2_online/templates/captura/egreso_form.html

```diff
@@ -0,0 +1,4 @@
+<a class="btn btn-primary" href="{% url 'captura:create_egreso' %}" id="create_estudio">
+    <i class="glyphicon glyphicon-plus"></i>
+    Agregar egreso
+</a>
```

Comment:
Are you sure we need to separate one button into a file by itself?? I think this add unnecessary complexity. The captura templates folders already has too many files . Besides... this is not a form, it is just a button.